src/QMCWaveFunctions/Jastrow/BsplineJastrowCuda.cu

```diff
@@ -25,134 +25,107 @@ bool AisInitialized = false;
 __constant__ float AcudaSpline[48];
 __constant__ double AcudaSpline_double[48];
 
-inline __device__  float  recipSqrt (float x)
-{
-  return rsqrtf(x);
-}
-inline __device__  double recipSqrt (double x)
-{
-  return rsqrt(x);
-}
+inline __device__ float recipSqrt(float x) { return rsqrtf(x); }
+inline __device__ double recipSqrt(double x) { return rsqrt(x); }
 
-inline __device__ float dist (float dx, float dy, float dz)
-{
-  return sqrtf(dx*dx + dy*dy + dz*dz);
-}
+inline __device__ float dist(float dx, float dy, float dz) { return sqrtf(dx * dx + dy * dy + dz * dz); }
 
-inline __device__ double dist (double dx, double dy, double dz)
-{
-  return sqrt(dx*dx + dy*dy + dz*dz);
-}
+inline __device__ double dist(double dx, double dy, double dz) { return sqrt(dx * dx + dy * dy + dz * dz); }
 
 
-void
-cuda_spline_init()
+void cuda_spline_init()
 {
-  float A_h[48] = { -1.0/6.0,  3.0/6.0, -3.0/6.0, 1.0/6.0,
-                    3.0/6.0, -6.0/6.0,  0.0/6.0, 4.0/6.0,
-                    -3.0/6.0,  3.0/6.0,  3.0/6.0, 1.0/6.0,
-                    1.0/6.0,  0.0/6.0,  0.0/6.0, 0.0/6.0,
-                    0.0,     -0.5,      1.0,    -0.5,
-                    0.0,      1.5,     -2.0,     0.0,
-                    0.0,     -1.5,      1.0,     0.5,
-                    0.0,      0.5,      0.0,     0.0,
-                    0.0,      0.0,     -1.0,     1.0,
-                    0.0,      0.0,      3.0,    -2.0,
-                    0.0,      0.0,     -3.0,     1.0,
-                    0.0,      0.0,      1.0,     0.0
-                  };
-  cudaMemcpyToSymbol(AcudaSpline, A_h, 48*sizeof(float), 0,
-                     cudaMemcpyHostToDevice);
-  double A_d[48] = { -1.0/6.0,  3.0/6.0, -3.0/6.0, 1.0/6.0,
-                     3.0/6.0, -6.0/6.0,  0.0/6.0, 4.0/6.0,
-                     -3.0/6.0,  3.0/6.0,  3.0/6.0, 1.0/6.0,
-                     1.0/6.0,  0.0/6.0,  0.0/6.0, 0.0/6.0,
-                     0.0,     -0.5,      1.0,    -0.5,
-                     0.0,      1.5,     -2.0,     0.0,
-                     0.0,     -1.5,      1.0,     0.5,
-                     0.0,      0.5,      0.0,     0.0,
-                     0.0,      0.0,     -1.0,     1.0,
-                     0.0,      0.0,      3.0,    -2.0,
-                     0.0,      0.0,     -3.0,     1.0,
-                     0.0,      0.0,      1.0,     0.0
-                   };
-  cudaMemcpyToSymbol(AcudaSpline_double, A_d, 48*sizeof(double), 0,
-                     cudaMemcpyHostToDevice);
+  float A_h[48] = {-1.0 / 6.0, 3.0 / 6.0, -3.0 / 6.0, 1.0 / 6.0, 3.0 / 6.0, -6.0 / 6.0, 0.0 / 6.0, 4.0 / 6.0,
```

Comment:
I think it would be worthwhile to preserve the formatting on the spline coefficients (here and A_d below) - they naturally work in groups of 4.